lib/testing/src/lib/core/pages/login.page.ts

```diff
@@ -163,7 +183,7 @@ export class LoginPage {
 
     goToLoginPage() {
         browser.waitForAngularEnabled(true);
-        browser.driver.get(TestConfig.adf.url + TestConfig.adf.port + '/login');
+        browser.driver.get(browser.baseUrl + '/login');
```

Comment:
in order for this to be generic, this should also consider the case when # is used (like in the case of ACA)